ert_storage/endpoints/experiments.py

```diff
@@ -3,6 +3,7 @@
 from sqlalchemy.orm.attributes import flag_modified
 from ert_storage.database import Session, get_db
 from ert_storage import database_schema as ds, json_schema as js
+from .priors import experiment_priors_to_dict
```

Comment:
Use absolute path import
```suggestion
from ert_storage.endpoints.priors import experiment_priors_to_dict
```